src/VkBootstrap.cpp

```diff
@@ -1547,7 +1547,11 @@ bool PhysicalDevice::enable_features_node_if_present(detail::GenericFeaturesPNex
     actual_pdf2.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_FEATURES_2;
     fill_chain.chain_up(actual_pdf2);
 
-    detail::vulkan_functions().fp_vkGetPhysicalDeviceFeatures2(physical_device, &actual_pdf2);
+    if (instance_version >= VKB_VK_API_VERSION_1_1) {
+        detail::vulkan_functions().fp_vkGetPhysicalDeviceFeatures2(physical_device, &actual_pdf2);
+    } else {
```

Comment:
```suggestion
    } else if (properties2_ext_enabled) {
```
Good catch! However, to be even more correct the physical device properties 2 extension must be enabled.

Reply:
Apologies for the sloppiness! I think this iteration should be complete